data/Language/Bash.yaml

```diff
@@ -1,5 +1,34 @@
 identifier: Bash
+full_name: Bash
 wikidata: Q189248
 extensions:
   - sh
+aliases:
+  - bash
 line_continuation: '\'
+delimiters:
+  # comment delimiters:
+  - hash
+  - bash_multiline_comment
+  # string delimiters:
+  - c_char
+  - backticks
+  - double_quote_slash_escape
+  - bash_single_quote_escape
+keywords:
+  - if
+  - then
+  - else
+  - elif
+  - fi
+  - case
+  - esac
+  - for
+  - select
+  - while
+  - until
+  - do
+  - done
+  - in
+  - function
+  - time
```

Comment:
This is a function, not a keyword.

Reply:
Hi @li-boxuan , I copied that from https://github.com/coala/coala/blob/master/coalib/bearlib/languages/definitions/Bash.py , should I add more *keywords*? Sorry if I misunderstood.

Reply:
Sorry for the misguidance. I just verified that `time` is indeed a keyword. Please ignore my previous review 😉 

ref: https://askubuntu.com/questions/445753/how-to-list-all-the-shell-keywords